alphagov/paas-s3-resource
# [#150512756] Bring the repository up to speed

## What

We're upgrading our concourse and we'd like to bring our forked resources up to speed at the same time.

In this PR we're merging the `upstream/master` with `gds_master`.

```sh
git pull
git co master
git pull upstream master
git co gds_master
git co -b 150512756-upgrade-concourse
git merge master
git push origin 150512756-upgrade-concourse
```

## How to review

- Check if our commit history has remained
- Check if we're up to date with `upstream`
- Check if my merge conflicts have been resolved properly - You can run `git diff upstream/master` and see relatively small amount of changes that does make sense and are covered by our commits

## Who can review

Neither @LeePorte nor @paroxp
